common/tools/sourcemap-path-remapper/src/index.ts

```diff
@@ -0,0 +1,139 @@
+// Sourcemap types!
+
+import * as fs from 'fs';
+import convertSourcemap from 'convert-source-map'; // Transforms sourcemaps among various common formats.
```

Comment:
~~I was mad at someone else but now I am mad at you :-P~~

```suggestion
import convertSourceMap from 'convert-source-map'; // Transforms sourcemaps among various common formats.
```